### PR TITLE
remove eclipse publish to p2 from pipeline

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: pipelines/pipelines/eclipse/build-branch.yaml [hunk 1]
```diff
@@ -248,47 +248,6 @@ spec:
       - --health
 #
```
Comment (eamansour) [inline pipelines/pipelines/eclipse/build-branch.yaml:249]: We'll also need to remove the `branch-docker-build-eclipse-p2` step in the pipeline to stop building the docker image for the p2 site

**Review by eamansour — APPROVED** [review]

File: pipelines/pipelines/eclipse/build-branch.yaml (hunk 1 above)
Comment (eamansour) [reply pipelines/pipelines/eclipse/build-branch.yaml:249]: Will be removed in a separate PR

Comment (KirbyKatcher) [reply pipelines/pipelines/eclipse/build-branch.yaml:249]: I will remove the` branch-docker-build-eclipse-p2` as part of the next PR which will come after I delete the docker images.

**Pull request merged**